### DISCUSSION TITLE
Windows OS Accessibility

### DISCUSSION THREAD

**Discussion by gooddavvy** [MonitorControl/MonitorControl#1400] [General]
Does MonitorControl support Windows OS? If not (yet), will it?

Just for your knowledge, I yet prefer Windows than Mac.

**Comment by waydabber** [comment]
Sorry, it does not and it won't. You might want to try TwinkleTray https://twinkletray.com

**Reply by gooddavvy** [reply]
It's okay. I will still get a Mac and try it one time.
